### ISSUE TITLE
Linking error in latest master.

### ISSUE THREAD

**Issue by rygo6** [microsoft/MixedReality-WebRTC#513]
Hello. I just merged the latest master and am trying to build x64 release, but I am getting this linking error:

```
Severity	Code	Description	Project	File	Line	Suppression State
Error	LNK2001	unresolved external symbol "class rtc::scoped_refptr<class webrtc::AudioDeviceModule> __cdecl webrtc::CreateWindowsCoreAudioAudioDeviceModule(void)" (?CreateWindowsCoreAudioAudioDeviceModule@webrtc@@YA?AV?$scoped_refptr@VAudioDeviceModule@webrtc@@@rtc@@XZ)	mrwebrtc-win32	C:\mrwebrtc\tools\build\mrwebrtc\win32\global_factory.obj	1	
```

I did run the command `.\build.ps1 -BuildConfig Release -BuildArch x64 -BuildPlatform Win32` and it finished successfully. 

What do you think this could be?

**Comment by djee-ms** [comment]
The `libwebrtc` build is missing the new ADM2 module. The ADM2 module for some reason is only built if the `libwertc` tests are built (and we are not building them by default); I reported [the bug](https://bugs.chromium.org/p/webrtc/issues/detail?id=11544) to Google some time ago, but in the meantime we need a local change to fix. I probably have it on my machine and it has not been pushed correctly, sorry about that. I will have a look now.

In the meantime you should be able to work around it by building the WebRTC tests; add the `--includeTests` options to the `run.py` script in [`build.ps1:110`](https://github.com/microsoft/MixedReality-WebRTC/blob/e839cd78859aa21f858839732e0f4f40d7178b20/tools/build/build.ps1#L110), which should I think build the tests, though I never tested that.

**Comment by djee-ms** [comment]
Fixed with #516, please update the branch and the submodules (`git submodule update --recursive`).
